src/main/java/interface_adapter/club_logged_in/ClubLoggedInState.java

```diff
@@ -11,10 +11,17 @@ public class ClubLoggedInState {
     private String email = "";
 
     private String password = "";
-    private String errorMessage;
+    private String message;
```

Comment:
What was the reason for this change?

Reply:
I made it so the club logged in view has a text label on the bottom of the page for any success or fail messages. I changed the name to reflect the potential of its use